lib/Metrc/client.rb

```diff
@@ -164,9 +163,11 @@ def destroy_plant_batches(license_number, resources)
     end
 
     def create_plant_batch_package(license_number, resources)
-      uri = '/plantbatches/v1/createpackages'
-
-      uri = '/plantbatches/v1/create/plantings' if NEW_PACKAGE_PLANTING_STATES.include?(configuration.state.to_sym)
+      uri = if configuration.state.to_sym == :ca
```

Comment:
why convert `to_sym`? what's wrong with simple old strings?

Reply:
Just in case. But I can change it back to string if you like.